lightpath/main.py

```diff
@@ -37,24 +54,38 @@ def main(db, hutches):
     db: str
         Path to happi JSON database
     """
+    if cfg:
+        logger.info(f'reading config from: {cfg}...')
+        with open(cfg, 'r') as f:
+            conf = yaml.safe_load(f)
+    else:
+        conf = {}
 
+    timeout = conf.get('timeout', 10)  # timeout (s)
```

Comment:
```suggestion
    timeout = float(conf.get('timeout', 10))  # timeout (s)
```
Better to bail early here if the config is invalid